eng/pipelines/common/templates/runtimes/build-test-job.yml

```diff
@@ -147,7 +147,7 @@ jobs:
     # Zip and publish managed test components
     - template: /eng/pipelines/common/upload-artifact-step.yml
       parameters:
-        rootFolder: $(managedTestArtifactRootFolderPath)
```

Comment:
Nit - should we redefine <code>managedTestArtifactRootFolderPath</code> rather than open-coding the <code>rootFolder</code> so that we don't get out of sync in other yml scripts?

Reply:
This is a single-use special case of this path, so I think we want it to be different. The "osGroup" parameter to this job is still not "AnyOS" as it needs to match the libraries build so we can correctly locate the artifacts, but we want to download this artifact into the correct folder in each other build (which is all of the other use cases of `$(managedTestArtifactRootFolderPath).

Reply:
I think that ultimately I would be happier if we could express this in a cleaner way, e.g. by introducing a new parameter "logicalOsGroup" that would be AnyOS in this case and propagating it all the way down to

https://github.com/dotnet/runtime/blob/16300e0aee3392bc604b258db0901d9047e80f7c/eng/pipelines/coreclr/templates/xplat-pipeline-job.yml#L113

but I guess we can offload that to a future cleanup task and possibly present is as an up-for-grabs issue for infra volunteers.

Reply:
In other words, I don't see this as blocking feedback but please consider creating the cleanup issue and adding it to the CoreCLR infrastructure backlog.

Reply:
I actually believe such a task to be very instructive for junior team members eager to ramp up on yml scripts.

Reply:
Tracking in https://github.com/dotnet/runtime/issues/61790